cmd/stack_package.go

```diff
@@ -146,6 +146,12 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 				return errors.Errorf("Error creating stacks directory: %v", err)
 			}
 
+			err = os.RemoveAll(stackPath)
```

Comment:
should we remove the one in line 158 `os.RemoveAll(stackPath)` and the `defer` in 163?

Reply:
Don't think so.  We shouldn't keep the packaging folder if possible.